frontend/common/src/components/form/RadioGroup/RadioGroup.stories.tsx

```diff
@@ -89,7 +81,7 @@ RadioGroupOfElements.args = {
 export const LargeRadioGroup = TemplateRadioGroup.bind({});
 LargeRadioGroup.parameters = {
   viewport: {
-    defaultViewport: "mobile1",
+    defaultViewport: "iphone6p",
```

Comment:
I tried so hard but could not figure out a way to type annotate the keys of INITIAL_VIEWPORTS and improve type safety.  For example, `keyof typeof INITIAL_VIEWPORTS` or `[key in INITIAL_VIEWPORTS]: Viewport`.  :shrug: